spec/integ/matrix-client-event-timeline.spec.js

```diff
@@ -540,20 +555,46 @@ describe("MatrixClient event timelines", function() {
             expect(timeline.getEvents().find(e => e.getId() === THREAD_REPLY.event_id)).toBeTruthy();
         });
 
-        it("should return undefined when event is not within a thread but timelineSet is", async () => {
+        it("should return relevant timeline from non-thread timelineSet when asking for the thread root", async () => {
```

Comment:
The current title is accurate, just trying avoid "non-thread" as we're really dealing with `timelineSet`'s which either represent a thread or the main timeline.

I think this is still accurate:

```suggestion
        it("should return relevant main timeline when event is a threadRoot given the main timelineSet (threadRoots belong to the main timeline and thread)", async () => {
```